icons.js

```diff
@@ -1,10 +1,15 @@
 const {Clutter,Gio,GLib,GObject,Shell,St} = imports.gi;
 
 var getIcon = function getIcon(playerAddress){
-	let icon = new St.Icon({
+	const Config = imports.misc.config;
```

Comment:
Import statements belong to the top (Line 2)

Reply:
agreed